packages/js/admin-e2e-tests/src/fixtures/http-client.ts

```diff
@@ -2,13 +2,15 @@
  * External dependencies
  */
 import { HTTPClientFactory } from '@woocommerce/api';
-/* eslint-disable @typescript-eslint/no-var-requires */
-const config = require( 'config' );
+import config from 'config';
 
 // Prepare the HTTP client that will be consumed by the repository.
 // This is necessary so that it can make requests to the REST API.
-const admin = config.get( 'users.admin' );
-const url = config.get( 'url' );
+const admin = config.get( 'users.admin' ) as {
+	username: string;
+	password: string;
+};
+const url = config.get( 'url' ) as string;
```

Comment:
@tomalec Instead of using type assertion, I think it would be great if we could just use the angle-bracket syntax since it's native-supported by `config` module.

```ts
const admin = config.get< {
	username: string;
	password: string;
} >( 'users.admin' );
const url = config.get< string >( 'url' );
```